Docs additions (#1)
nest-desktop/nest-desktop-docs: troubleshootings/error.rst

.. |see| image:: /_static/img/icons/arrow-right.svg
   :alt: See
   :height: 17.6px
   :target: #

.. _gateway-time-out:

Gateway Time-out
----------------
This means that the NEST Server did not receive a response from the NEST
Simulator within a certain period of time.

If you are using `NEST Desktop on EBRAINS <https://nest-desktop.apps.hbp.eu>`__:
Our setup on EBRAINS is not ready for bigger simulations and may lead to this
error if it takes too long.

Please reduce the load, e.g. by reducing the population size or
the simulating time. It also helps to use a bigger simulation resolution.

.. image:: /_static/img/screenshots/ebrains-504-error.png
   :target: #ebrains-504-error

|

.. _problem-with-firefox-private-mode:

Firefox private mode problems
-----------------------------

When NEST Desktop is opened in a private window in Firefox,
only a blank screen is shown. This is due to an internal

> configuration in Firefox. If you are a very experienced user,

Reviewer: ```suggestion
configuration in Firefox. If you are an experienced user,
```

> you might have have a look at

Reviewer: ```suggestion
you might have a look at
```
